tools/cli/src/sourcemaps/upload.ts

```diff
@@ -346,11 +360,23 @@ function pipeAssets(assets: AssetWithContent<RawSourceMapWithDebugId>[], include
     };
 }
 
-function getUploadUrl(opts: Partial<UploadOptions>): Result<string | undefined, string> {
+export function getUploadUrl(opts: Partial<UploadOptions>): Result<string | undefined, string> {
+    if (opts.url && opts.subdomain) {
+        return Err('--url and --subdomain are exclusive');
+    }
+
     if (opts.url) {
         return validateUrl(opts.url);
     }
 
+    if (opts.subdomain) {
+        if (!opts.token) {
+            return Err('token is required with subdomain');
+        }
+
+        return Ok(`https://submit.backtrace.io/${opts.subdomain}/${opts.token}/sourcemap`);
```

Comment:
I think its worth to note the URL won't work on the on prem environment. What do you think about putting this note in notes?

Reply:
Done